flood_forecast/model_dict_function.py

```diff
@@ -18,7 +22,7 @@
 "TransformerXL": TransformerXL, "DummyTorchModel": DummyTorchModel,  "LSTM":LSTMForecast, "SimpleLinearModel": SimpleLinearModel, 
 "CustomTransformerDecoder":CustomTransformerDecoder}
 
-pytorch_criterion_dict = {"MSE": MSELoss(), "SmoothL1Loss":SmoothL1Loss(), "PoissonNLLLoss":PoissonNLLLoss()}
+pytorch_criterion_dict = {"MSE": MSELoss(), "SmoothL1Loss":SmoothL1Loss(), "PoissonNLLLoss": PoissonNLLLoss(), "RMSELoss": RMSE, "MAPE": MAPELoss()}
```

Comment:
Change RMSE to RMSELoss()

Reply:
Yeah saw that. Should've waited for unit tests before posting. Thanks.